### ISSUE TITLE
Lack documentation

### ISSUE THREAD

**Issue by tkdchen** [onepiecejs/nodejs-cantas#73]

**Comment by xiaods** [comment]
we need define next roadmap, this is urgent purpose.

**Comment by tkdchen** [comment]
right. milestone v1.1 could be rescheduled for next release.

**Comment by xiaods** [comment]
@tkdchen yeah. currently i working on another project, need you take the leadership right now.
